### PR TITLE
[Submission] Zai El Amri, Tautz, Ritter, Melnik

### PR DESCRIPTION
**Pull request by wzaielamri**
**AUTHOR**

Dear mdx workshop committee,

I request a review for the following MDX submission:

### **Title:**  Transfer Learning with Jukebox for Music Source Separation  

**Author(s):**  Wadhah Zai El Amri, Oliver Tautz, Helge Ritter, Andrew Melnik  

**Category *** Audio signal processing, Neural networks, Transfer learning, Jukebox  

* [x] `POSTERS` _(+Abstract: 250 Words+one page pdf)_
  > As a participant of the MDX challenge, please know that your poster would be automatically accepted after a minimal prescreening. Hence, don't forget to provide us with the link to your team in the AIcrowd website.
  > You are also welcome to submit us some unrelated abstract, in case your interest would be in presenting some method or idea you would just like to discuss with the participants passing by.  
  > => You *don't* need to include your poster in the PR. Just be ready to have it for D-day and show it to the audience through screen sharing.
AIcrowd Link: https://www.aicrowd.com/challenges/music-demixing-challenge-ismir-2021/teams/citec_uni_bielefeld  
* [ ] `LONG TALK`
  > A long talk will last about 20-30min, including questions. This can be about some recent paper you published, or may also present some original research and investigations you did. If you pick this category, please provide us with an extended abstract of the talk, or directly with the slides if you have them available, even in an unfinished state, at your convenience. 
* [ ] `DISCUSSIONS`
  > This category is intended to stimulate fruitful discussions and collaborations. If you pick it, it means you are willing to chair some open discussion with the participants, mainly around one particular aspect of music demixing research. For that purpose, you should provide a minimal abstract as well as a list of a few questions you would like the participants to discuss about. Don't hesitate to propose ideas, We will possibly contact you in case we identify several similar topics that were proposed!

**Preferred Timezone***

* [x] A 11:00 PM (PDT) to 9:00 AM PDT
* [ ] B 11:00 AM (PDT) to 9:00 PM PDT

  > The preferred timezone will help us to organize the day schedule. You should select at least one, but can perfectly select both if you don't care. We will organize the programme so that to maximally fit your preferences.
---

**ORGANISATION COMMITTEE (do not fill out)**

* [x] Editor acknowledgment
* [x] Reviewer 1 
* [ ] Reviewer 2
* [x] Review 1 decision [accept]
* [ ] Review 2 decision [accept/reject]
* [ ] Editor decision [accept/reject]

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -1,139 +1,91 @@
...
 ---
 
 # Abstract
 
-The abstract should be around 250 words long as usual, and should be provided
-for submissions of all categories: posters, long talks and discussions.
-* Please briefly describe the particular points your submission focuses on and/or the
-problems it aims to solve.
-* For all categories except discussions, please also briefly summarize the contributions.
+In this paper, we demonstrate a neural network architecture that uses representations of a publicly available pretrained _Jukebox_ model and transfer learning to solve the problem of audio source separation. Jukebox takes 3 days of training on 256 GPUs. In this work, we demonstrate how to adapt Jukebox's audio representations for the problem of extraction of an audio source from a single mixed audio channel. Results demonstrate competitive performance to the other state-of-the-art approaches. Our approach is fast to train. We provide an open-source code implementation of our architecture.
```
Comment (aliutkus) [inline paper.md:28]: * the problem of extracting an
* I wouldn't say your results demonstrate competitive performance, but are definitely inspiring

File: paper.md [hunk 2]
```diff
@@ -1,139 +1,91 @@
...
-
-
-# Example of content fitting the template
+\begin{equation}
+\label{eq:1}
+y_t=D(argmin(\|E_1(x_t)- c)\|) \;\; \text{for} \;\; c \in C
+\end{equation}
 
-## Introduction
+describes a full forward pass through the VQ-VAE, where $ y_t $ is the prediction for an input sequence $x_t$ and $\|.\|$ is the euclidean norm. For further technical details on the used VQ-VAE architecture, refer to the paper of Dhariwal et al. [@dhariwal2020jukebox]. The model is fine-tuned on data for one stem, learning good representations for a single instrument. In addition, we train a second encoder $E_2$, identical to the one already mentioned, to project an input sequence of the mixture to the space already known by the codebook and decoder. For deployment, the encoder of the VQ-VAE is switched with the second one, effectively mapping from the full mixture to one stem.
```
Comment (aliutkus) [inline paper.md:53]: I think the mixture should look different to the stems in your figures. This was a bit confusing to me, it basically looks like these are stems everywhere, and not mixtures sometimes

**Review by aliutkus — COMMENTED** [review]
a demonstration of the applicability of the jukebox to source separation. The method amounts to learn a stem-specific autoencoder, and then learn an encoder that maps the mixture to each stem's code.
The particularity is that the model considered is some recently proposed huge beast from the litterature

File: paper.md (hunk 2 above)
Comment (wzaielamri) [reply paper.md:53]: Thank you for your grateful feedback

**Comment by aliutkus** [comment]
To all participants:
please note that you should update your submission by November 6th 23:00 UTC

Thank you very much

**Comment by wzaielamri** [comment]
@aliutkus I already updated the submission based on your review. 
Thank you for your feedback :)

**Comment by faroit** [comment]
@wzaielamri Great to have you presenting a poster. I couldn't find you among the workshop registrants at ISMIR. Please register within the next day so that I can add you to the schedule.

**Comment by wzaielamri** [comment]
> @wzaielamri Great to have you presenting a poster. I couldn't find you among the workshop registrants at ISMIR. Please register within the next day so that I can add you to the schedule.

@faroit Ihope i correctly registered myself to the: "Music Demixing Challenge Satellite Registration (ISMIR attendees)"
Can another member of my team attend instead of me?

**Comment by faroit** [comment]
> > @wzaielamri Great to have you presenting a poster. I couldn't find you among the workshop registrants at ISMIR. Please register within the next day so that I can add you to the schedule.
> 
> @faroit Ihope i correctly registered myself to the: "Music Demixing Challenge Satellite Registration (ISMIR attendees)" Can another member of my team attend instead of me?

yes, please make sure you send me who will be the presenting author ahead of time

**Comment by wzaielamri** [comment]
> > > @wzaielamri Great to have you presenting a poster. I couldn't find you among the workshop registrants at ISMIR. Please register within the next day so that I can add you to the schedule.
> > 
> > 
> > @faroit Ihope i correctly registered myself to the: "Music Demixing Challenge Satellite Registration (ISMIR attendees)" Can another member of my team attend instead of me?
> 
> yes, please make sure you send me who will be the presenting author ahead of time

should all the team members register? or not important?

**Comment by faroit** [comment]
> > > > @wzaielamri Great to have you presenting a poster. I couldn't find you among the workshop registrants at ISMIR. Please register within the next day so that I can add you to the schedule.
> 
> > > 
> 
> > > 
> 
> > > @faroit Ihope i correctly registered myself to the: "Music Demixing Challenge Satellite Registration (ISMIR attendees)" Can another member of my team attend instead of me?
> 
> > 
> 
> > yes, please make sure you send me who will be the presenting author ahead of time
> 
> 
> 
> @faroit  should all the team members register? or not important?

Just the one who presents is fine

**Comment by wzaielamri** [comment]
@faroit  how long should the presentation of a poster be and which materials are required?  (slides?)

**Comment by faroit** [comment]
@wzaielamri the poster session is 1h long and presentations can be about to be 5min long. The poster will be a single page presentation that you share via screen share. You can use a pdf to scroll in the PDF. then will usually lead to spontaneous discussion and talks with the attendees. We will send out more information within the next day.

**Comment by faroit** [comment]
👏 The submission has been published on midspace (the ISMIR conference platform). The submissions will also be hosted on the workshop website tomorrow. 

The schedule can be seen there or [here](https://docs.google.com/spreadsheets/d/1PaksFcHWUfNPMg-XloA3CJtrvb9uMU0erCFVyvFi9B4/edit?usp=sharing). Please let us know if there is anything missing. Otherwise we see each other on Friday/Saturday!

**Comment by faroit** [comment]
@wzaielamri we haven't heard from you regarding the poster presentation tomorrow. Did you receive the reminder for submitting the poster file? Is everything set for tomorrow?

**Comment by faroit** [comment]
@wzaielamri please update us one the poster presentation today

**Comment by wzaielamri** [comment]
@faroit I emailed you.

**Pull request closed without merging**